Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -206,6 +215,37 @@ private static async Task<string> FindMsBuildPathAsync()
             return string.Empty;
         }
 
+        private static bool UpdateVSProj(IBuildInfo buildInfo)
+        {
+            // For ARM64 builds we need to add ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch
+            // to vcxproj file in order to ensure that the build passes
+
+            string projectName = PlayerSettings.productName;
+            string projectFilePath = $"{Path.GetFullPath(buildInfo.OutputDirectory)}\\{projectName}\\{projectName}.vcxproj";
+
+            if (!File.Exists(projectFilePath))
+            {
+                Debug.LogError($"Cannot find project file: {projectFilePath}");
+                return false;
+            }
+
+            var rootNode = XElement.Load(projectFilePath);
+            var defaultNamespace = rootNode.GetDefaultNamespace();
+            var propertyGroupNode = rootNode.Element(defaultNamespace + "PropertyGroup");
+            
+            if (propertyGroupNode == null)
+            {
+                propertyGroupNode = new XElement(defaultNamespace + "PropertyGroup", new XAttribute("Label", "Globals"));
+                rootNode.Add(propertyGroupNode);
+            }
+
+            propertyGroupNode.Add(new XElement(defaultNamespace + "ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch", "None"));
```

Comment:
Do you know if running this twice will cause it to add two elements (which are both this thing?)

I think when I did the capability thing (i.e. gaze input) I had to check to see if it existed before adding it, otherwise it would actually add it twice.